### DISCUSSION TITLE
Aligning text to top left in a table

### DISCUSSION THREAD

**Discussion by MelB507** [community/community#50671] [New to GitHub] [answered]
Helllooo. 
I'm a newbie using GitHub.  Is it possible to align text to the top left of a cell in a table?  I know how to align left, right & center, just not to the top of the cell.

Any help would be much appreciated.
Ta
Mel

**Comment by Majid-J** [comment, marked as answer]
Hey!
I believe you are refering to Markdown tables? In that case you can only left-align, center or right-align the content of a cell. For more information you can visit: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

**Comment by Anmol-Baranwal** [comment]
It is not possible to vertically align text to the top of a cell in a Markdown table on GitHub. This is because Markdown doesn't provide a way to control the vertical alignment of content within a cell.

While you can use HTML and CSS to style the table, including aligning text to the left, right, or center, vertical alignment is not possible using Markdown on GitHub.

To align, you can use:
```
Left-
| Header 1 | Header 2 |
|:----|:----|   

Center-
| Header 1 | Header 2 |
|:---:|:---:|  
```

**Comment by MelB507** [comment]
Booooooooooo - ok thanks guys

**Reply by Wesseldr** [reply]
You can simulate the top allign with using the html br code in the cell you like to push the line up like this:

| | |
|-|-|
| **OS** | Debian 12 Bookworm
| **Databases**<br><br> | <ul><li>SalesDB</li><li>StockDB</li></ul>

**Reply by iandall** [reply]
I don't know why this works, but it does!

**Comment by cdauth** [comment]
While this does not seem to be possible using Markdown syntax, it is possible using HTML syntax inside Markdown and using `<td valign="top">` for the table cells:

```markdown
<table>
<tbody>
<tr>
<td>

**OS**

</td>
<td>Debian 12 Bookworm</td>
</tr>
<tr>
<td valign="top">

**Databases**

</td>
<td>

* SalesDB
* StockDB

</td>
</tr>
</tbody>
</table>
```

<table>
<tbody>
<tr>
<td>

**OS**

</td>
<td>Debian 12 Bookworm</td>
</tr>
<tr>
<td valign="top">

**Databases**

</td>
<td>

* SalesDB
* StockDB

</td>
</tr>
</tbody>
</table>

Unfortunately, if you want to use Markdown inside the table cells, you must insert two newlines after `<td>` and again two newlines before `</td>`. This makes the Markdown look really ugly, but at least the result looks nice.
